versions.yml

```diff
@@ -18,17 +18,17 @@
 ---
 SourceType: "standalone-build-scripts"
 Revisions:
-  - Revision: HEAD
```

Comment:
Could you please instead create a branch and update versions.yml on the branch.

Reply:
Using llvm-15 branch